src/words/Words.js

```diff
@@ -0,0 +1,151 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
To be honest, I can't go through this file, since it is more complex than the others and I'm short on time right now. It looks good to me though